cmake/ycm-0.2/find-modules/FindZFP.cmake

```diff
@@ -0,0 +1,54 @@
+# FindZFP
+# ----------------
+# Find the ZFP includes and library.
+# Once done this will define the following variables::
+
+# ZFP_INCLUDE_DIR     - ZFP include directory
+# ZFP_LIBRARY         - ZFP libraries 
+# ZFP_FOUND 					- if false, you cannot build anything that requires ZFP
+
+#=============================================================================
+# Copyright 2016 iCub Facility, Istituto Italiano di Tecnologia
+#   Authors: Nicolo' Genesio <nicolo.genesio@iit.it>
+#
+# Distributed under the OSI-approved BSD License (the "License");
+# see accompanying file Copyright.txt for details.
+#
+# This software is distributed WITHOUT ANY WARRANTY; without even the
+# implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the License for more information.
+#=============================================================================
+# (To distribute this file outside of YCM, substitute the full
+#  License text for the above reference.)
+
+include(FindPackageHandleStandardArgs)
+
+
+if(WIN32)
+	set(CMAKE_DEBUG_POSTFIX "d") 
```

Comment:
The `CMAKE_DEBUG_POSTFIX` variable has a precise meaning in CMake . Please use another variable name if you want to use this variable only locally in this module.